README.md

```diff
@@ -22,8 +22,6 @@ We welcome contributions! Please see our [contribution guide](https://github.com
 
 Deprecation Warning
 -------------------
-The 1.0.0 release of ipfx brings some new features, like NWB2 support, along with improvements to our documentation and testing. We will also drop support for
-- NWB1
-- Python 2
 
-Older versions of ipfx will continue to be available, but may receive only occasional bugfixes and patches.
+The 1.1.0 release of IPFX drops support for Python 3.6 which reached end of life and stopped receiving security updated on December 23, 2021.
```

Comment:
If we're following Semantic Versioning rules, IPFX should be on 2.0.0 after these changes because we are dropping Python3.6 support.

Reply:
Sounds good.